applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -16,7 +16,20 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_DVFS_LOG_LEVEL);
 
-#include <ld_dvfs_handler.h>
+#include <zephyr/devicetree/clocks.h>
```

Comment:
And why do we need it here?

Reply:
You explicitly use `DT_ALIAS` macro here. Because of that you should include corresponding header without relying on fact that another header would include it for you

Reply:
we often relay on some include files to include others. I can add it if you insist but do not feel it is a mistake to leave it as it is.

Reply:
It is generally a mistake as you directly use `DT_ALIAS` macro which is not guaranteed to be provided by `zephyr/devicetree.h`. You should not rely on some other header to pull in the needed header.
If e.g. at some point the `nrf_clock_control.h` stops bringing in the `zephyr/devicetree.h`, our build would fail.

Reply:
An additional include would not hurt here (to ensure that needed macros are provided)